Misc/NEWS.d/next/Documentation/2019-12-09-10-12-00.bpo-39002.AfgvfO.rst

```diff
@@ -0,0 +1 @@
+Fix simple typo in Lib/test/test_statistics.py.
```

Comment:
For fixing simple types, we don't need to add `NEWS` entries. Can you delete it?

Reply:
thx Done